include/DirectoryManager.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+#define PATH_MAX 4096
+constexpr auto ERROR_UNKNOWN_DIR = "Unknown Directory";
+constexpr auto ERROR_UNSPECIFIED_DIR = "cd: Directory not specified\n";
+constexpr auto ERROR_CHANGE_DIR = "cd: Failed to change directory\n";
+
+class DirectoryManager
```

Comment:
This is more a matter of style, but if a class has only public attributes and methods maybe it should be a struct